composer.json

```diff
@@ -9,6 +9,7 @@
         "cweagans/composer-patches": "~1.6",
         "commerceguys/intl": "~1.0",
         "drupal/apigee_edge": "~1.0",
+        "drupal/requirement": "1.x-dev",
```

Comment:
We shouldn't be adding a dev version of a module. Let's use a 1.0 version. I know this is on rc1 still, anything missing to move it to 1.0? Also, need to update that project with https://github.com/apigee/requirement-drupal project codebase before adding.